### PR TITLE
Add several missing "license.md" files (especially for docker-library images)

### PR DESCRIPTION
**Pull request by tianon**
Ref #1046

### CODE REVIEW

File: crux/license.md [hunk 1]
```diff
@@ -0,0 +1 @@
+View [license information](TODO) for the software contained in this image.
```
Comment (tianon) [inline crux/license.md]: This is still a `TODO`. :innocent:

File: mysql/license.md [hunk 2]
```diff
@@ -0,0 +1 @@
+View [license information](https://www.mysql.com/about/legal/) for the software contained in this image.
```
Comment (tianon) [inline mysql/license.md]: @ltangvald is this an appropriate place to link for MySQL license information? (it's non-obvious, but seems to be the best I could find in a quick search)

File: mongo-express/license.md [hunk 3]
```diff
@@ -0,0 +1 @@
+View [license information](https://github.com/mongo-express/mongo-express#license) for the software contained in this image.
```
Comment (tianon) [inline mongo-express/license.md]: @knickers does this look correct?

File: neurodebian/license.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+View [license information](https://www.debian.org/social_contract#guidelines) for the software contained in this image.
+
+Additionally, `-nonfree` flavors of the images enable access to materials which are not fully DFSG-compliant, and might have restrictions (e.g. non-commercial) and/or lack original sources.
```
Comment (tianon) [inline neurodebian/license.md]: @yarikoptic LGTY?

Comment (yarikoptic) [reply neurodebian/license.md]: IANAL, FWIW LGTM

**Comment by tianon** [comment]
@knickers @ltangvald feel free to PR or poke me post-merge if these aren't right or could be better :heart: :+1:

**Pull request merged**